src/main.cpp

```diff
@@ -430,15 +416,38 @@ static std::tuple<string, string> _create_cert(const Options &options, const str
 		csr = _spawn("openssl req -new -config " + cnf + " -key " + pkey_file);
 		return std::make_tuple(pkey, csr);
 	} else {
+		// Initialize the HSM and create a key in it.
+		_setenv("OPENSSL_CONF", cnf.c_str());
+
+		/* Just in case we are using a TPM */
+		_setenv("TPM2_PKCS11_STORE", "/var/tpm2_pkcs11");
```

Comment:
Isn't this path already set as the default tpm2 pkcs11 store? While useful for development, this will end up causing the store path to be maintained in two places, which is not ideal.